showhidden.fish

```diff
@@ -0,0 +1,11 @@
+function showhidden -d "Hides/reveals system files and folders in Finder"
+  if test "yes" = (echo $argv | tr '[:upper:]' '[:lower:]')
```

Comment:
A `switch` statement might be simpler:

```
switch "$argv"
case Yes YES yes
    ...
case No NO no
    ...
case '*'
    echo "Command input must be 'yes' or 'no'"
end
```

Ref: http://fishshell.com/docs/current/tutorial.html#tut_conditionals